core/src/apps/base.py

```diff
@@ -43,6 +43,20 @@ def busy_expiry_ms() -> int:
     return expiry_ms if expiry_ms > 0 else 0
 
 
+def _language_version_matches() -> bool | None:
+    """
+    Whether translation blob version matches firmware version.
+    Returns None if there is no blob.
+    """
+    from trezor import translations
+
+    header = translations.TranslationsHeader.load_from_flash()
+    if header is None:
+        return None
```

Comment:
I would lean towards returning `True` here
semantically, "language is en-US and it does not need update"

Reply:
Fixed in [a6dfe95](https://github.com/trezor/trezor-firmware/pull/3544/commits/a6dfe9558cdcdba2723d39a7b59b90fe1674e7b5), this was actually the reason for the failing test.

Reply:
but now you also need to update the test :)

Reply:
uhhhh yeah 3215107b2da53aba9207586677f55404f3443c19